package.json

```diff
@@ -16,7 +17,8 @@
     "accessible-autocomplete": "^2.0.4",
     "cypress": "11.2.0",
     "govuk-frontend": "^4.4.0",
-    "jquery": "^3.6.1"
+    "jquery": "^3.6.1",
+    "tinymce": "^5.10.7"
```

Comment:
Paste plugin is no longer free on v6 of tinymce so installed the latest v5 release. Version 5.10 will be supported until April 2023 so might need to think about paying for tinymce (to use the paste plugin, which has been renamed to powerpaste) or find an alternative text editor. I will put this all in a ticket